packages/devp2p/src/protocol/protocol.ts

```diff
@@ -0,0 +1,83 @@
+import ms from 'ms'
+import { debug as createDebugLogger } from 'debug'
+import { EventEmitter } from 'events'
+import { Peer, DISCONNECT_REASONS } from '../rlpx/peer'
+
+export class Protocol extends EventEmitter {
+  _peer: Peer
+  _statusTimeoutId: NodeJS.Timeout
+  _message_codes: any
```

Comment:
let's stick to camelCase and name this `_messageCodes`

Reply:
Good idea. I've been jumping between different style standards.